### PR TITLE
SPOI-10221 Added walkthrough document for Kinesis to S3 application template

### PR DESCRIPTION
**Pull request by chaithu14**
@yogidevendra Please review and merge.

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@chaithu14 . Please change the physical tab image (it points to monitor tab) also verify with Ashwin about properties specification as per new support.

**Comment by chaithu14** [comment]
Updated the PR with physical tab change and properties change as per schema UI.

**Comment by deepak-narkhede** [comment]
Looks fine to me. Will wait for Yogi's comments.

**Comment by yogidevendra** [comment]
Screenshot of launch dialogue should include launch button.

**Comment by yogidevendra** [comment]
Apart from this, it looks OK.
@deepak-narkhede Feel free to merge this once above comment is incorporated.

**Comment by chaithu14** [comment]
@yogidevendra Launch button is there in screenshot of launch dialogue which is in disabled state. This should be enabled when the mandatory properties specified.

File: docs/app-templates/kinesis-to-s3.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    |Secret Key For Kinesis Input|KINESIS_SECRET_KEY|
+    |End Point For Kinesis Input|KINESIS_END_POINT|
+    |Access Key For S3Output|S3_ACCESS_KEY|
+    |Secret Access Key For S3Output|S3_SECRET_KEY|
+    |S3Output Bucket Name|S3_BUCKET_NAME|
+    |S3Output Directory Path|/user/appuser/output|
+
+    Details about configuration options are available in [Configuration options](#configuration_options) section.
+
+1. Click on the `Launch` button on lower right corner of the dialog to launch the application.
```
Comment (yogidevendra) [inline docs/app-templates/kinesis-to-s3.md]: This refers to property-editor.png screenshot in this case

**Comment by yogidevendra** [comment]
@chaithu14  Please add Launch button to property-editor.png

File: docs/app-templates/kinesis-to-s3.md (hunk 1 above)
Comment (chaithu14) [inline docs/app-templates/kinesis-to-s3.md]: Done.

**Comment by yogidevendra** [comment]
@deepak-narkhede Could you please verify if things looks OK with mkdocs serve. You can go ahead and merge the PR if everything is fine.

**Comment by deepak-narkhede** [comment]
Verified with mkdocs.

**Pull request merged**